src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml

```diff
@@ -33,39 +37,53 @@
                 <div class="col l-4">
                     <div class="item patient-item1">
                         <div>
-                            <img src="~/Image/no-avatar.jpg" alt="Avatar" class="patient-img" />
+                            @if (item.AvatarLink == "")
+                            {
+                                <img src="~/Image/no-avatar.jpg" alt="Avatar" class="patient-img" />
+                            }
+                            else
+                            {
+                                <img src="@item.AvatarLink" alt="Avatar" class="patient-img" />
+                            }
                         </div>
                         <div class="patient-name">
-                            <p class="patient-name-bold">@item.FullName</p>
+                            <p class="fs-6 fw-bold">@item.FullName</p>
                             <p>@item.PatientIdentifier</p>
                         </div>
                     </div>
                 </div>
                 <div class="col l-4">
                     <div class="item patient-item2">
                         <div>
-                            <p class="dob-bold">DoB</p>
-                            <p>@item.DoB</p>
+                            <p>DOB</p>
+                            <p class="fw-bold">@item.DoB</p>
                         </div>
                         <div>
-                            <p class="dob-bold">Gender</p>
-                            <p>@item.Gender</p>
+                            <p>Gender</p>
+                            <p class="fw-bold">@item.Gender</p>
                         </div>
                     </div>
                 </div>
                 <div class="col l-4">
                     <div class="item patient-item3">
                         <div>
                             <div class="contact-icon">
-                                <i class="fas fa-phone"></i>
+                                <i class="fas fa-phone fa-rotate-90"></i>
                             </div>
                             <p class="contact-info">@item.PhoneNumber</p>
                         </div>
                         <div>
                             <div class="contact-icon">
                                 <i class="fas fa-envelope"></i>
                             </div>
-                            <p class="contact-info">@item.Email</p>
+                            @{
+                                if (item.Email != null)
+                                {
+                                    string[] result = item.Email.Split("@");
+                                    <p class="email-link">@result[0]</p>
+                                    <p class="contact-info">@@@result[1]</p>
```

Comment:
Why this has 3 '@' ?

Reply:
the first two @@ is for display @ in razor code block, and the last one is for transition result[1] variable to HTML content. Please review this @vietvo361